data/mods/Magiclysm/snippets/newspapers.json

```diff
@@ -6,6 +6,10 @@
       {
         "id": "magiclysm_one_year_old_news_1",
         "text": "Yesterday Gregory Hamilton, the local Wizard Enforcer, went on a rampage in the local shopping district.  Known for being the most powerful mage in Syracuse, New York, it is unsurprising that he managed to reduce the entire district into a smoking crater.  The National Guard took him out with a team of snipers and 5 helicopters.  Casualties are unknown, but current estimates are 15,000 and climbing."
+      },
+      {
+        "id": "magiclysm_one_year_old_news_1",
+        "text": "Scientists from the only magical research facility in the region reported a breakthrough yesterday.  They reported that their research lead, Ander McDowell, has \"Ascended\" to a greater plane of existence.  They are still looking into the details, because once Ander did so, they only made one statement before appearing to dissolve into the Ether: \"I've found the answer! It's been Mana all along!\""
```

Comment:
```suggestion
        "text": "Scientists from the only magical research facility in the region reported a breakthrough yesterday.  They reported that their research lead, Ander McDowell, has \"Ascended\" to a greater plane of existence.  They are still looking into the details, because once Ander did so, they only made one statement before appearing to dissolve into the Ether: \"I've found the answer!  It's been Mana all along!\""
```